src/tests/unit/onbase-dao.test.js

```diff
@@ -1,33 +1,79 @@
-const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
-const proxyquire = require('proxyquire');
-const sinon = require('sinon');
-
-const conn = require('../../api/v1/db/oracledb/connection');
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import config from 'config';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+import createError from 'http-errors';
```

Comment:
This line should be right after `import config from 'config';` ([alphabetical order](https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#importing-modules)).